amlab.js

```diff
@@ -0,0 +1,111 @@
+console.log('question1');
+let arr1 = [1, 2, 3, 4, 5];
+let result1 = arr1.map((el) => {
+  return el + 10;
+});
+console.log(result1);
+
+console.log('question2');
+let arr2 = ["a", 123, "b", "c", {name: "cat"}];
+let strings2 = arr2.map((el) => {
+ if(typeof el !== 'string') {
+   return '';
+ } else {
+   return el;
+ }
+});
+console.log(strings2);
+console.log('question3');
+let arr7a = [1, 2, 3, 4, 5];
+let filtered = arr7a.filter((el) => el % 2 === 0);
+console.log(filtered);
+
+console.log('question4');
+let arr4 = [1,1,1,1,2];
+let result4 = arr4.every(el => {
+  if (el === arr4[0]) {
+    return true;
+  }
+
+  return false;
+});
+
+console.log(result4);
+
+console.log('question 5');
+
+function adder(total, el) {
+  return total + el;
+}
+
+let nums = [1, 2, 3, 4, 5];
+let total = nums.reduce((total, el) => {
+return total + el;
+
+}, 0)
+console.log(total);
```

Comment:
Great job! There is a shorter and cleaner way ...

function adder(arr) {
	  return arr.reduce((total, currentElem) => total + currentElem);
	}